Makefile

```diff
@@ -0,0 +1,33 @@
+# Includes the respective Makefile depending on the OS
+# Have a look at either posix.mak, win32.mak or win64.mak for a list of available commands.
+
+DMD_DIR=../dmd
+include $(DMD_DIR)/src/osmodel.mak
```

Comment:
The Phobos Makefiles assume the existence of a DMD directory (see e.g. `posix.mak`).

Reply:
Hmm. I looked at `dmd/src/osmodel.mak` but it only seems to detect Posix OS flavors, there doesn't seem to be anything that detects windows?

Reply:
I believe @quickfur is right. I had a look at `dmd/test/Makefile` and it seems to have the Windows checks written aside from including `dmd/src/osmodel.mak`. Maybe `osmodel.mak` needs an update?